x/ibc/02-client/genesis.go

```diff
@@ -0,0 +1,28 @@
+package client
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// InitGenesis initializes the ibc client submodule's state from a provided genesis
+// state.
+func InitGenesis(ctx sdk.Context, k Keeper, gs GenesisState) {
+	for _, client := range gs.Clients {
+		k.SetClientState(ctx, client)
+		k.SetClientType(ctx, client.GetID(), client.ClientType())
+	}
+	for _, cs := range gs.ClientsConsensus {
+		for _, consState := range cs.ConsensusStates {
+			// TODO: double-check if the consState will be invalidated on import at 0 height.
+			k.SetClientConsensusState(ctx, cs.ClientID, consState.GetHeight(), consState)
+		}
+	}
```

Comment:
@cwgoes @AdityaSripal what happens in the case of a zero height export? should I just set one consensus state (the one w/ latest height) at height 0?

Reply:
This is a client of another chain, why would we change the logic on a zero-height export?

In any case, we should stop doing zero-height exports anyways once IBC is operational.